clientapi/clientapi_test.go

```diff
@@ -2330,8 +2330,23 @@ func TestCreateRoomInvite(t *testing.T) {
 
 		roomID := gjson.GetBytes(w.Body.Bytes(), "room_id").Str
 		validRoomID, _ := spec.NewRoomID(roomID)
+
+		// Confirm that the room matches the private state preset
+		ev, err := rsAPI.CurrentStateEvent(context.Background(), *validRoomID, spec.MRoomJoinRules, spec.Invite)
+		if err != nil {
+			t.Fatal(err)
+		}
+		ev, err = rsAPI.CurrentStateEvent(context.Background(), *validRoomID, spec.MRoomHistoryVisibility, string(gomatrixserverlib.HistoryVisibilityShared))
+		if err != nil {
+			t.Fatal(err)
+		}
+		ev, err = rsAPI.CurrentStateEvent(context.Background(), *validRoomID, spec.MRoomGuestAccess, "can_join")
+		if err != nil {
+			t.Fatal(err)
+		}
```

Comment:
Looks like we need to check the content of the state events here, as currently we don't use `ev` at all. (except the check if bob has a `displayname` below)